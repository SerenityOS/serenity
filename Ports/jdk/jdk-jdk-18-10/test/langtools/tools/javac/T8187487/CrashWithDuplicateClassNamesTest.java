/*
 * @test /nodynamiccopyright/
 * @bug 8187487
 * @summary crash with duplicate class name
 * @compile/fail/ref=CrashWithDuplicateClassNamesTest.out -XDrawDiagnostics CrashWithDuplicateClassNamesTest.java
 */

class CrashWithDuplicateClassNamesTest {
    static class C1$C2 {}

    static class C1 {
        static class C2 {}
    }

    static class C3 {
        static class C4 {}
    }

    static class C3$C4 {}
}
