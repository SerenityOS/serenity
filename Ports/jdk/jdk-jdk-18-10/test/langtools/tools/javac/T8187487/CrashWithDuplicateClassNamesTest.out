CrashWithDuplicateClassNamesTest.java:12:16: compiler.err.same.binary.name: C1$C2, C2
CrashWithDuplicateClassNamesTest.java:19:12: compiler.err.same.binary.name: C4, C3$C4
2 errors
